src/bowling.js

```diff
@@ -0,0 +1,89 @@
+import Frame from './frame.js';
+
+export default class Bowling {
+  constructor() {
+    this.frames = [];
+    this.gameStatus = true;
+    this.totalScore = 0;
+    this._newFrame();
+  }
+
+  roll(pins) {
+    if (!this.gameStatus) return;
+    this.currentFrame.roll(pins, this._isBonus);
```

Comment:
Is the current frame always the last one in `this.frames`? If yes, then storing it separately is duplication of information. You could have a method `this.currentFrame()` that returns the last element of `this.frames`.